### PR TITLE
Adds JS Libs for autocomplete and tokenize

### PR DESCRIPTION
**Pull request by tbiScireum**

### CODE REVIEW

File: src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+                    return;
+                }
+
+                $.each(this.rows, function (index, row) {
+                    var template = completions.templateForType(row.type);
+
+                    var $row = $(Mustache.render(template, row));
+                    completions.addHoverRowEvent($row);
+                    completions.element.append($row);
+                });
```
Comment (meg1502) [inline src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta]: after rendering, we need to call lazyload on the images inside the dropdown. I see no event or other possiblity to take this action after every update

Comment (tbiScireum) [reply src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta]: -> events.afterRender() & events.afterRenderRow()

File: src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+                completions.update();
+            },
+
+            /**
+             * Will handle key up events that should not be handled as input change but as something else.
+             *
+             * @@return boolean true if the event was handled, else false
+             */
+            onNonInputKeys: function (event) {
+                if (event.keyCode === sirius.keys.KEY_ENTER) {
```
[the remark this answers is not in the document]
Comment (tbiScireum) [reply src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta]: done

File: src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+                    completions.selectedRow = undefined;
+                    this.state = this.VISIBLE;
+                    this.update();
+                    completions.element.show();
+                    events.onShow();
+                }
+            },
+
+            hide: function () {
+                if (completions.state === completions.HIDDEN) {
```
[the remark this answers is not in the document]
Comment (tbiScireum) [reply src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta]: done

**Pull request merged**